action: add zran-no-prefetch benchmark and make smoke.yml tidy

Desiki-high opened the pull request:
- Benchmark in smoke test has too many same steps, we can move them to shell and make [smoke.yml](https://github.com/dragonflyoss/image-service/tree/master/.github/workflows/smoke.yml) tidy by prepare_env.sh and benchmark_summary.sh.
- Add zran-no-prefetch benchmark.
- Change the benchmark-result order, set the oci in the first.
- Since https://github.com/dragonflyoss/image-service/pull/1238, we move the benchmark-result from PR comment to action summary, we should enable benchmark-result in push and schedule.
- Use the stable wordpress tag 6.11.
- Delete the artifacts after benchmark-result download all artifacts. Certainly, the nydus-integration-test had downloaded the artifacts.

imeoer commented on .github/workflows/smoke.yml:
```diff
@@ -148,23 +146,12 @@ jobs:
-        sudo docker push localhost:5000/${{env.IMAGE}}:latest
-        git clone https://github.com/magnific0/wondershaper.git
-        sudo install -D -m 755 wondershaper/wondershaper /usr/local/bin
+        sudo install -D -m 755 misc/benchmark/prepare_env.sh prepare_env.sh
```
No need to install the script.

Desiki-high replied:
fix

imeoer commented on .github/workflows/smoke.yml:
```diff
@@ -193,32 +180,13 @@ jobs:
-    - name: Prepare Nydus Registry Environment
-      run: |
-        sudo docker run -d --restart=always -p 5000:5000 --name registry registry
+        sudo install -D -m 755 misc/benchmark/prepare_env.sh prepare_env.sh
```
Ditto.

Desiki-high replied:
fix

imeoer commented on .github/workflows/smoke.yml:
```diff
@@ -247,32 +215,51 @@ jobs:
+          --target localhost:5000/${{env.IMAGE}}:${{env.TAG}}_nydus \
+          --fs-version 6 \
+          --oci-ref \
+          --platform linux/amd64,linux/arm64
```
It seems we just only run the test on `linux/amd64` platform, `linux/arm64` is unnecessary.

Desiki-high replied:
> It seems we just only run the test on `linux/amd64` platform, `linux/arm64` is unnecessary.

Actually, we don't need to declaration the platform, so I delete the opt.

Desiki-high committed:
misc: delete unused benchmark code

Signed-off-by: Desiki-high <ding_yadong@foxmail.com>

Desiki-high committed:
misc: create shell for benchmark

1. prepare_env.sh for prepare container environment.
2. benchmark_summary.sh for benchmark-result to summary result.

Signed-off-by: Desiki-high <ding_yadong@foxmail.com>

Desiki-high committed:
action: add zran-no-prefetch in benchmark

1. add the zran without prefetch benchmark.
2. move the same steps to prepare_env.sh.
3. move benchmark summary script to benchmark_summary.sh.
3. change the benchmark-result order and enable in push and schedule.
4. set stable the wordpress tag 6.1.1.
5. delete the artifacts after benchmark-result download all artifacts.

Signed-off-by: Desiki-high <ding_yadong@foxmail.com>
